api/uniter/unit_test.go

```diff
@@ -618,6 +618,34 @@ func (s *unitSuite) TestWatchTrustConfigSettings(c *gc.C) {
 	notifyWatcher.AssertOneChange()
 }
 
+func (s *unitSuite) TestWatchTrustConfigSettingsHash(c *gc.C) {
+	watcher, err := s.apiUnit.WatchTrustConfigSettingsHash()
+	c.Assert(err, jc.ErrorIsNil)
+
+	stringsWatcher := watchertest.NewStringsWatcherC(c, watcher, s.BackingState.StartSync)
+	defer stringsWatcher.AssertStops()
+
+	// Initial event.
+	stringsWatcher.AssertChange("92652ce7679e295c6567a3891c562dcab727c71543f8c1c3a38c3626ce064019")
```

Comment:
can we comment that this the hash of the empty bson struct as done elsewhere

Reply:
Yup, will do!